pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -1791,3 +1801,80 @@ func (c *Cluster) releaseOrchestrationLock() {
 	c.orchestrationMutex.Unlock()
 	logger.Debugf("Released lock for mon orchestration")
 }
+
+func (c *Cluster) RotateMonCephxKeys() (bool, error) {
+	shouldRotateMonKeys, err := shouldRotateMonKeys(c.context, c.ClusterInfo)
+	if err != nil {
+		return false, errors.Wrapf(err, "failed to check if cephx keys for mon daemons in the namespace %q should be rotated", c.ClusterInfo.Namespace)
+	}
+
+	if !shouldRotateMonKeys {
+		logger.Debugf("cephx key rotation for mon daemon in the namespace %q is not required", c.ClusterInfo.Namespace)
+		return shouldRotateMonKeys, nil
+	}
+
+	logger.Infof("cephx keys for mon daemons in the namespace %q will be rotated", c.ClusterInfo.Namespace)
+
+	k := keyring.GetSecretStore(c.context, c.ClusterInfo, c.ClusterInfo.OwnerInfo)
+	newKey, err := k.RotateKey("mon.")
```

Comment:
why it is not visible in `ceph auth ls` ?

Reply:
Not sure. Need to check with Patrick